migration/migration.go

```diff
@@ -16,23 +17,35 @@ import (
 	"github.com/NethermindEth/juno/encoder"
 	"github.com/NethermindEth/juno/utils"
 	"github.com/bits-and-blooms/bitset"
+	"github.com/fxamacker/cbor/v2"
 	"github.com/sourcegraph/conc/pool"
 )
 
+var (
+	schemaVersionKey           = db.SchemaVersion.Key()
+	schemaIntermediateStateKey = db.SchemaVersion.Key([]byte{1})
```

Comment:
We should add the new key to `db/buckets.go`. Otherwise we may overwrite this on accident.